mesh/mesh.hpp

```diff
@@ -815,6 +845,13 @@ class Mesh
    void AddHexAsTets(const int *vi, int attr = 1);
    void AddHexAsWedges(const int *vi, int attr = 1);
    void AddHexAsPyramids(const int *vi, int attr = 1);
+   void AddHexAs24TetsWithPoints(int *vi,
+                                 std::map<std::tuple<int, int, int, int>, int>
+                                 &hex_face_to_center,
+                                 int attr = 1);
+   void AddQuadAs4TrisWithPoints(int *vi, int attr = 1);
+
+   void AddQuadAs5QuadsWithPoints(int *vi, int attr = 1);
```

Comment:
I appreciate that the other methods do not have it so there's a question of consistency, but I think these should have doxygen comments. If you're game, feel free to add to the other methods to achieve consistency as that would be much appreciated, but only on your newly introduced methods would be sufficient for this PR.